planner/core/plan_cacheable_checker.go

```diff
@@ -231,27 +231,19 @@ func NonPreparedPlanCacheableWithCtx(sctx sessionctx.Context, node ast.Node, is
 	}
 	from := selectStmt.From
 	if from == nil || selectStmt.From.TableRefs == nil {
-		return false, "queries that have sub-queries are not supported"
+		return false, "queries that have sub-queries are not supported zzzz"
```

Comment:
```suggestion
		return false, "queries that have sub-queries are not supported"
```
this `zzzz`may not be expected